Zend/tests/gh7771_1.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+GH-7771 (Incorrect file/line for class constant expression exceptions)
+--FILE--
+<?php
+
+include __DIR__ . '/gh7771_1_definition.inc';
+
+new Foo();
+
+?>
+--EXPECTF--
+Fatal error: Uncaught Error: Class "NonExistent" not found in %s/Zend/tests/gh7771_1_definition.inc:4
+Stack trace:
+#0 {main}
+  thrown in %s/Zend/tests/gh7771_1_definition.inc on line 4
```

Comment:
```
========DIFF========
001+ Fatal error: Uncaught Error: Class "NonExistent" not found in C:\projects\php-src\Zend\tests\gh7771_1_definition.inc:4
001- Fatal error: Uncaught Error: Class "NonExistent" not found in %s/Zend/tests/gh7771_1_definition.inc:4
     Stack trace:
     #0 {main}
004+   thrown in C:\projects\php-src\Zend\tests\gh7771_1_definition.inc on line 4
004-   thrown in %s/Zend/tests/gh7771_1_definition.inc on line 4
========DONE========
FAIL GH-7771 (Incorrect file/line for class constant expression exceptions) [C:\projects\php-src\Zend\tests\gh7771_1.phpt] 
PASS get_class_methods(): Testing scope [C:\projects\php-src\Zend\tests\get_class_methods_001.phpt] 
                                                      
========DIFF========
001+ Fatal error: Uncaught Error: Class "NonExistent" not found in C:\projects\php-src\Zend\tests\gh7771_2_definition.inc:6
001- Fatal error: Uncaught Error: Class "NonExistent" not found in %s/Zend/tests/gh7771_2_definition.inc:6
     Stack trace:
     #0 {main}
004+   thrown in C:\projects\php-src\Zend\tests\gh7771_2_definition.inc on line 6
004-   thrown in %s/Zend/tests/gh7771_2_definition.inc on line 6
========DONE========
```

Other than the platform dependence of the tests the tests and implementation look good, my review comments were addressed. (Appveyor Windows build failed)

```suggestion
  thrown in %sgh7771_1_definition.inc on line 4
```

Reply:
Thanks, I always forget about windows :stuck_out_tongue: